src/crimson/os/cyan_store.cc

```diff
@@ -4,6 +4,228 @@
 
 #include "common/safe_io.h"
 
+#include "crimson/os/cyan_collection.h"
+#include "crimson/os/cyan_object.h"
+#include "crimson/os/Transaction.h"
+
+namespace {
+  seastar::logger& logger() {
+    return ceph::get_logger(ceph_subsys_filestore);
+  }
+}
+
+namespace ceph::os {
+
+using ObjectRef = boost::intrusive_ptr<Object>;
+
+CyanStore::CyanStore(const std::string& path)
+  : path{path}
+{}
+
+CyanStore::~CyanStore() = default;
+
+seastar::future<> CyanStore::mount()
+{
+  bufferlist bl;
+  string fn = path + "/collections";
+  string err;
```

Comment:
I am not sure if  path+/collections  file is a little big, seastar thread should think the read_file block or not ( we didn't return a future for the file read, right?)  did you test it?

Reply:
for the same reason, we can always rewrite it if it's slow.